src/resources/court_spec/schemas/court_spec_json_example.txt

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
this file should name as `court-spec-example.json`

Reply:
update